cachi2/interface/cli.py

```diff
@@ -400,6 +419,8 @@ def merge_sboms(
         help="Names of files with SBOMs to merge.",
     ),
     output_sbom_file_name: Optional[Path] = OUTFILE_OPTION,
+    sbom_type: SBOMFormat = SBOM_TYPE_OPTION,
+    sbom_name: Optional[str] = typer.Option(None, "--sbom-name", help="Name of the merged SBOM."),
```

Comment:
The help string is a bit unclear. What do you think about "Name of the resulting merged SBOM as it appears in SBOM."?